x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleInitialisationTests.java

```diff
@@ -364,6 +368,38 @@ public void testMasterFailover() throws Exception {
         });
     }
 
+    public void testCreatePolicyOnClose() throws Exception {
```

Comment:
This name is a little weird, maybe:

```suggestion
    public void testCreatePolicyWhenStopped() throws Exception {
```